src/dswx_sar/defaults/algorithm_parameter_s1.yaml

```diff
@@ -134,8 +136,10 @@ runconfig:
 
         inundated_vegetation:
             enabled: True
-            temporal_avg_path:
-            initial_class_path:
+            vv_vh_ratio_max: 12
+            vv_vh_ratio_min: 7
```

Comment:
Just wondering if there are any case that the product is in HH / HV polarizations. If that's the case, how about renaming it to sth. like `cross_pol_ratio_max` and `cross_pol_ration_min`?